src/cmake/GeosxMacros.cmake

```diff
@@ -54,3 +54,36 @@ macro( geosx_add_code_checks )
     endif()
 
 endmacro( geosx_add_code_checks )
+
+##------------------------------------------------------------------------------
+## geos_add_test( NAME       [name]
+##                COMMAND    [command]
+##                EXECUTABLE [executable] )
+##
+## Adds a test to the project, remaining arguments are forwarded to `blt_add_test`
```

Comment:
Can you mention which global property you are using (`geos_tests_exe_list`) in the doc please?